### DISCUSSION TITLE
vaex.ml train_test_split() - split by class percentages

### DISCUSSION THREAD

**Discussion by AkshataDM** [vaexio/vaex#2357] [Q&A]
Hello, it sounds like you can achieve randomization with vaex.ml train_test_split() using shuffle on the dataset. But, i'm looking for stratified sampling on train test splits. Has this been considered before?
